addons/hr_timesheet/views/project_portal_templates.xml

```diff
@@ -3,12 +3,12 @@
 
     <template id="portal_my_task" inherit_id="project.portal_my_task" name="Portal: My Task with Timesheets">
         <xpath expr="//li[@id='task-nav']" position="before">
-            <li t-if="timesheets and allow_timesheets" class="list-group-item flex-grow-1" id='nav-report'>
+            <li t-if="timesheets and allow_timesheets" class="list-group-item flex-grow-1 d-flex flex-wrap justify-content-center align-items-center" id='nav-report'>
                 <div class="o_download_pdf btn-toolbar flex-sm-nowrap">
-                    <div class="btn-group flex-grow-1 me-1 mb-1">
+                    <div class="btn-group flex-grow-1 mr-1">
```

Comment:
`mr` does not exist anymore since we are using bootstrap 5, it has been replace by `me`. I guess that `mr-1` does not have any effects now so I guess you can remove it? Could you test?